docs/deployment-local-cluster.md

```diff
@@ -0,0 +1,46 @@
+# Deployment on Local Cluster
+
+This project allows you to spin up virtualized Kubernets/OpenShift cluster. In
+this guide we will create a local Kubernetes cluster with two nodes and preinstalled
+node dependencies, then we will deploy kubernetes-nmstate from local sources.
+
+If you want to deploy kubernetes-nmstate on your arbitrary cluster, go to
+[deployment on arbitrary cluster guide](deployment-arbitrary-cluster.md).
+
+Start local cluster. If you want to use OpenShift instead of Kubernetes or
+different amount of nodes, check [development guide](developer-guide.md#local-cluster).
+
+```shell
+KUBEVIRT_NUM_NODES=2 make cluster-up
+```
+
+Build kubernetes-nmstate from local sources and install it on the cluster.
+(Please note that this will not start kubernetes-nmstate controllers on the
+cluster, for more information, read
+[active vs. on-demand](user-guide-active-vs-on-demand.md))
+
+```shell
+make cluster-sync
+```
+
+You can ssh into created nodes using `cluster/cli.sh`.
+
+```shell
+cluster/cli.sh ssh node01
+```
+
+Finally you can access Kubernetes API using `cluster/kubectl.sh`.
+
+```shell
+./cluster/kubectl.sh get nodes
+```
+
+If you want to use `kubectl` to access the cluster, start proxy.
```

Comment:
start *the* proxy. or start *a* proxy - if there is really a proxy being started. Not sure if this should forwards the port even though it is called proxy.

Reply:
Done